src/main/kotlin/com/launchdarkly/intellij/FlagStore.kt

```diff
@@ -11,6 +11,7 @@ import com.launchdarkly.api.ApiException
 import com.launchdarkly.api.model.FeatureFlag
 import com.launchdarkly.api.model.FeatureFlags
 import com.launchdarkly.intellij.coderefs.FlagAliases
+/* ktlint-disable no-unused-imports */
```

Comment:
Added an upstream comment on an open issue: https://github.com/pinterest/ktlint/issues/1277#issuecomment-1047938411

Without this disable line the import on line 15 will be removed.